tests/testsuite/artifact_dep.rs

```diff
@@ -20,7 +20,7 @@ fn check_with_invalid_artifact_dependency() {
                 version = "0.0.0"
                 authors = []
                 resolver = "2"
-                
```

Comment:
my text editor automatically removes trailing whitespaces, hope that is okay... I actually think this would be a good idea to add to `rustfmt` :slightly_smiling_face: